src/docs/background/off-session-payments/index.md

```diff
@@ -62,7 +83,7 @@ kicks in), the payment card might have expired, etc. It is therefore
 always important to allow for a fall-back payment path in your
 transaction process. Since only one transition from a state can be
 triggered automatically, this fall-back payment path must be defined to
-trigger upon a user action.
+trigger upon a user action, as in the example.
 
 You can build upon this example and extend it to make the payment
 process more robust. For instance, in case the customer fails to pay for
```

Comment:
Feedback welcome: Would it make sense to have a section on 'what to pay attention to if you implement a process like this in FTW'? I can make those additions as well - not necessarily on a tutorial-specific level but as a general 'good to be aware' listicle.

Reply:
I think a listicle sounds like a good idea! The more information is available here, the less likely it is that people will target related questions at support.

Reply:
First attempt at listicle written, appreciate any feedback!